packages/web/components/drawers/drawer.tsx

```diff
@@ -57,7 +57,7 @@ export const Drawer: FunctionComponent<{
         props.onClose?.();
       },
     }),
-    [isOpen, isAnimationComplete, setIsOpen]
+    [isOpen, isAnimationComplete, setIsOpen, props]
   );
```

Comment:
Won't this memo run every render if the props object is passed? My thought is that if React triggers a re-render due to props changing, then the memo would run every render since props is in deps... LMK if I'm off here.

Reply:
You're right, since React uses referential equality in dependency arrays and I believe props is not the same referential object across re-renders. I'll destructure props and pass them in instead.

Reply:
You're right, since React uses referential equality in dependency arrays and I believe props is not the same referential object across re-renders. I'll destructure props and pass them in instead.

Reply:
You're right, since React uses referential equality in dependency arrays and I believe props is not the same referential object across re-renders. I'll destructure props and pass them in instead.

Reply:
You're right, since React uses referential equality in dependency arrays and I believe props is not the same referential object across re-renders. I'll destructure props and pass them in instead.

Reply:
You're right, since React uses referential equality in dependency arrays and I believe props is not the same referential object across re-renders. I'll destructure props and pass them in instead.

Reply:
You're right, since React uses referential equality in dependency arrays and I believe props is not the same referential object across re-renders. I'll destructure props and pass them in instead.

Reply:
You're right, since React uses referential equality in dependency arrays and I believe props is not the same referential object across re-renders. I'll destructure props and pass them in instead.

Reply:
You're right, since React uses referential equality in dependency arrays and I believe props is not the same referential object across re-renders. I'll destructure props and pass them in instead.

Reply:
You're right, since React uses referential equality in dependency arrays and I believe props is not the same referential object across re-renders. I'll destructure props and pass them in instead.

Reply:
You're right, since React uses referential equality in dependency arrays and I believe props is not the same referential object across re-renders. I'll destructure props and pass them in instead.

Reply:
You're right, since React uses referential equality in dependency arrays and I believe props is not the same referential object across re-renders. I'll destructure props and pass them in instead.

Reply:
You're right, since React uses referential equality in dependency arrays and I believe props is not the same referential object across re-renders. I'll destructure props and pass them in instead.

Reply:
You're right, since React uses referential equality in dependency arrays and I believe props is not the same referential object across re-renders. I'll destructure props and pass them in instead.

Reply:
You're right, since React uses referential equality in dependency arrays and I believe props is not the same referential object across re-renders. I'll destructure props and pass them in instead.

Reply:
You're right, since React uses referential equality in dependency arrays and I believe props is not the same referential object across re-renders. I'll destructure props and pass them in instead.

Reply:
You're right, since React uses referential equality in dependency arrays and I believe props is not the same referential object across re-renders. I'll destructure props and pass them in instead.

Reply:
You're right, since React uses referential equality in dependency arrays and I believe props is not the same referential object across re-renders. I'll destructure props and pass them in instead.

Reply:
You're right, since React uses referential equality in dependency arrays and I believe props is not the same referential object across re-renders. I'll destructure props and pass them in instead.

Reply:
You're right, since React uses referential equality in dependency arrays and I believe props is not the same referential object across re-renders. I'll destructure props and pass them in instead.

Reply:
You're right, since React uses referential equality in dependency arrays and I believe props is not the same referential object across re-renders. I'll destructure props and pass them in instead.

Reply:
You're right, since React uses referential equality in dependency arrays and I believe props is not the same referential object across re-renders. I'll destructure props and pass them in instead.